US-CBP/cbp-theme
# Separate Tab Content

#### What's this PR do?
- Separates tab content within each page
- Spacing changes for the Get Started page
- Lightened grey color for tab

#### Where should the reviewer start?
- `typography.js`, `layouts.js`, `grids.js`, `tables.js`, `getStarted.js`
#### How should this be manually tested?
- Manually navigate through each existing page to check that the tab is directing to the right content

#### Any background context you want to provide?
- Need to adjust the right margin of the grey line for tabs
- Will delete tab component & existing references after PR review

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
